### ISSUE TITLE
TypeOrm cross platform version compatibility

### ISSUE THREAD

**Issue by jasonhealy** [uport-project/veramo#373]
In the past we have had to fix `@veramo/data-store` to use `typeorm@0.2.24` as `v0.2.25` caused issues. Since then our dependabot automated upgrade has bumped the version to `v0.2.30` which hasn't caused issues in node but has broken compatibility for React Native.  We need to asses to the pro's and cons of: 

- Fixing the version to support all platforms
- Installing an older version specifically for React Native
- Make `typeorm` a peer dependency so users will need to manually install it

Reference issue: https://github.com/uport-project/veramo-website/pull/41#issuecomment-774512727

To summarise the temporary solution for React Native until we have a more robust fix:

- Remove package-lock.json if it appears in your directory (it should only appear if you run npm vs yarn)
- Manually install `typeorm@0.2.24`
- Add a `resolutions` key to your `package.json` to ensure it uses this version

```
{
   "resolutions": {
      "typeorm": "0.2.24"
   }
}
```

**Comment by mirceanis** [comment]
This also relates to #372

**Comment by trentlarson** [comment]
Regarding this:

>     * Remove package-lock.json if it appears in your directory (it should only appear if you run npm vs yarn)

... note that the command `yarn add @zxing/text-encoding` from the instructions contains the following in the output:

`npm notice created a lockfile as package-lock.json. You should commit this file.`

**Comment by jasonhealy** [comment]
@trentlarson you can ignore this output as the project uses yarn. Adding that shim should be removed soon hopefully. https://github.com/Gozala/web-encoding/issues/9#issuecomment-768765160
